src/rules/parser.go

```diff
@@ -339,6 +348,11 @@ func (p *parser) parseRuleInfo(st ir.Node, labelStmt ir.Node, proto *Rule) (Rule
 	return rule, nil
 }
 
+func isURL(str string) bool {
+	re := regexp.MustCompile(`^((https?|ftp)://)?[^\s/$.?#].\S*$`)
```

Comment:
the regular expression is definitely not correct.

Reply:
After discussion, I removed it completely